src/tests/state/experiments/selectors.test.js

```diff
@@ -11,3 +11,84 @@ describe("Selectors", () => {
     expect(getExperimentsCount(mockedState)).toEqual(4);
   });
 });
+
+describe("getFilteredExperimentsByDate tests", () => {
+  it("should call getFilteredExperimentsByDate and return the first item in items", () => {
```

Comment:
Is that what `getFilteredExperimentsByDate` does? It always returns the first item in the list?

Reply:
Also the 'thing' we're talking about here is `getFilteredExperimentsByDate` and the individual test names describe what `it` (the function `getFilteredExperimentsByDate`) does, so does `getFilteredExperimentsByDate` `call getFilteredExperimentsByDate`? To me that suggests that it's a recursive function?